### PR TITLE
docs: Update azure READMEs with template contents

### PR DESCRIPTION
**Pull request by Josmithr**
Adds `Contribution Guidelines`, `Help`, and `Trademark` sections to package READMEs, and adds missing package entry to mono-repo README.

### CODE REVIEW

File: azure/README.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ The azure-client package provides a simple and powerful way to consume collabora
 
 See the [package README](./packages/azure-client/README.md) for more information.
 
+## @fluidframework/azure-local-services
+
+The azure-local-services package provides a minimal, self-contained, test implementation of the Azure Fluid Relay service that can be run locally and used for development/testing Fluid functionality in conjunction with the `AzureClient` in local mode.
+
+See the [package README](./packages/azure-client/README.md) for more information.
```
Comment (tylerbutler) [inline azure/README.md:13]:
```suggestion
See the [package README](./packages/azure-local-service/README.md) for more information.
```

File: azure/README.md [hunk 2]
```diff
@@ -6,6 +6,12 @@ The azure-client package provides a simple and powerful way to consume collabora
 
 See the [package README](./packages/azure-client/README.md) for more information.
 
+## @fluidframework/azure-local-services
+
+The azure-local-services package provides a minimal, self-contained, test implementation of the Azure Fluid Relay service that can be run locally and used for development/testing Fluid functionality in conjunction with the `AzureClient` in local mode.
```
Comment (tylerbutler) [inline azure/README.md:11]:
```suggestion
The azure-local-service package provides a minimal, self-contained, test implementation of the Azure Fluid Relay service that can be run locally and used for development/testing Fluid functionality in conjunction with the `AzureClient` in local mode.
```

File: azure/README.md [hunk 3]
```diff
@@ -6,6 +6,12 @@ The azure-client package provides a simple and powerful way to consume collabora
 
 See the [package README](./packages/azure-client/README.md) for more information.
 
+## @fluidframework/azure-local-services
```
Comment (tylerbutler) [inline azure/README.md:9]:
```suggestion
## @fluidframework/azure-local-service
```

**Review by tylerbutler — APPROVED** [review]

File: azure/README.md (hunk 1 above)
Comment (Josmithr) [reply azure/README.md:13]: 🤦‍♂️

**Pull request merged**